trieste/models/gpflow/models.py

```diff
@@ -646,6 +670,32 @@ def update(self, dataset: Dataset) -> None:
         num_data = dataset.query_points.shape[0]
         self.model.num_data = num_data
 
+    def get_inducing_variables(self) -> Tuple[TensorType, TensorType, TensorType, bool]:
+        """
+        Return the model's inducing variables.
+
+        :return: Tensors containing: the inducing points (i.e. locations of the inducing
+            variables); the variational mean q_mu; the Cholesky decomposition of the
+            variational covariance q_sqrt; and a bool denoting if we are using whitened
+            or not whitened representations.
+        """
+        inducing_variable = self.model.inducing_variable
+
+        if type(inducing_variable) in [
```

Comment:
Use isinstance as (1) it's more Pythonic, (2) mypy can infer things from it. Note that the second argument can be a tuple or list of types if you need more than one.

```suggestion
        if isinstance(inducing_variable, gpflow.inducing_variables.SharedIndependentInducingVariables):
```